### PR TITLE
Add README.md to some packages

### PR DESCRIPTION
**Pull request by imoans**
- Added README to all packages
- Added an overview and usage of frequently used packages

### CODE REVIEW

File: modules/redux/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+```js
+import { createMiddleware, reducer } from "@phenyl/redux";
+import { createStore, applyMiddleware } from "redux";
+import PhenylHttpClient from "@phenyl/http-client";
+
+const httpClient = new PhenylHttpClient({ url: "localhost:8080" });
+const middleware = createMiddleware({ client: httpClient });
+const enhancer = applyMiddleware(middleware);
+const store = createStore(phenylReducer, enhancer);
```
Comment (imoans) [inline modules/redux/README.md:17]:
```suggestion
const store = createStore(reducer, enhancer);
```

File: modules/central-state/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# @phenyl/central-state
+
+
+```js
+import { MongoClient } from 'mongodb'
```
Comment (sasurau4) [inline modules/central-state/README.md:5]: This code seems mongo-db example. We can't use central-state itself without db client.
Empty example would be better, I think.

**Review by sasurau4 — CHANGES_REQUESTED** [review]
With nits

**Review by sasurau4 — APPROVED** [review]
👍

**Pull request merged**